### ISSUE TITLE
GNB Burst (NoMercy windows)

### ISSUE THREAD

**Issue by wrldfx** [FFXIV-CombatReborn/RebornRotations#105]
**![image](https://github.com/user-attachments/assets/3728d611-5fc8-44b8-b4f0-4bb6e5003ef7)

This is my opener(cd 10) with gnb, the new lvl 100 skills is not being used, also they need a little configuration to not break the gnashing fang combo, since both can break eachother.
Also the rotation keep using 1-2-3 combo in NoMercy window.
Outside of NoMercy the 1-2-3 combo is overcapping the 3 bullets gauge.

New 100 stuff:

Burst strike is now used as a filler to not overcap the gauge (maybe in 2min burst if <2.46 gcd)
The new 3 skill combo can break Gnashing combo and vice versa.

Basically gnb now need to do in NoMercy window:

Gnashing
DD
Savage Claw
Wicked Talon
Reign of Beast (if 2 min or bloodfest nomercy)
Noble Blood
Lion Heart
Sonic Break.

Dont need to be in order, just dont breaking gnashing or reign combo is great.
Like said in Discord, im trying to learn the code to test some stuff, but ill make an issue to be documented here.
